docs/content/quatrix.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Quatrix"
+description: "Rclone docs for Quatrix"
+versionIntroduced: "v1.63.2"
+---
+
+# {{< icon "fas fa-shield-alt" >}} Quatrix
+
+Quatrix by Maytech is [Quatrix Secure Compliant File Sharing | Maytech](https://www.maytech.net/products/quatrix-business).
+
+Paths are specified as `remote:path`
+
+Paths may be as deep as required, e.g., `remote:directory/subdirectory`.
+
+The initial setup for Quatrix involves getting an API Key from Quatrix. You can get the API key in the user's profile at `https://<account>/profile/api-keys`
+or with the help of the API - https://docs.maytech.net/quatrix/quatrix-api/api-explorer#/API-Key/post_api_key_create.
+
+See complete Swagger documentation for Quatrix - https://docs.maytech.net/quatrix/quatrix-api/api-explorer
+
+## Configuration
+
+Here is an example of how to make a remote called `remote`.  First run:
+
+     rclone config
+
+This will guide you through an interactive setup process:
+
+```
+No remotes found, make a new one?
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> remote
+Type of storage to configure.
+Choose a number from below, or type in your own value
+[snip]
+XX / Quatrix by Maytech
+   \ "quatrix"
+[snip]
+Storage> quatrix
+API key for accessing Quatrix account.
+api_key> jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
```

Comment:
I hope this isn't a valid API key?!

I'm guessing it isn't but it might be best to change the string to `your_api_key` or something like that.